webapp/channels/src/components/admin_console/access_control/modals/user_sync/user_sync_modal.scss

```diff
@@ -0,0 +1,60 @@
+.user-list-modal-dialog {
```

Comment:
Similar to the policy text modal, this probably aren't needed if the modal has the `more-modal` class on it

Reply:
Updated 👍